### PR TITLE
Add atom language plugin to docs

### PR DESCRIPTION
**Pull request by joshka**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: share/doc/homebrew/Tips-N'-Tricks.md [hunk 1]
```diff
@@ -136,3 +136,9 @@ inline patches in Vim.
 [homebrew-mode](https://github.com/dunn/homebrew-mode) provides syntax
 highlighting for inline patches as well as a number of helper functions
 for editing formula files.
+
+### Atom
+
+[language-hombrew-formula](https://atom.io/packages/language-hombrew-formula)
```
Comment (MikeMcQuaid) [inline share/doc/homebrew/Tips-N'-Tricks.md]: Why is `hombrew` not spelt correctly?

File: share/doc/homebrew/Tips-N'-Tricks.md [hunk 2]
```diff
@@ -136,3 +136,9 @@ inline patches in Vim.
...
+adds highlighting and diff support (with the
```
Comment (MikeMcQuaid) [inline share/doc/homebrew/Tips-N'-Tricks.md]: Capitalise the `Adds`

File: share/doc/homebrew/Tips-N'-Tricks.md (hunk 1 above)
Comment (joshka) [inline share/doc/homebrew/Tips-N'-Tricks.md]: Typo was on the package and has now been fixed https://github.com/josa42/atom-language-homebrew-formula/issues/3#issuecomment-240013203

**Comment by MikeMcQuaid** [comment]
Feels like we should also link what this was based on e.g. https://github.com/samueljohn/Homebrew-formula-syntax

File: share/doc/homebrew/Tips-N'-Tricks.md (hunk 2 above)
Comment (joshka) [inline share/doc/homebrew/Tips-N'-Tricks.md]: Nope. This is correct as it matches the sentence. It's likely the diff doesn't make this obvious.

Comment (MikeMcQuaid) [inline share/doc/homebrew/Tips-N'-Tricks.md]: I see 👍

**Comment by joshka** [comment]
That's already there under the sublime heading.

File: share/doc/homebrew/Tips-N'-Tricks.md (hunk 1 above)
Comment (MikeMcQuaid) [inline share/doc/homebrew/Tips-N'-Tricks.md]: Another `fomula` typo in the GitHub readme, FYI.

**Comment by MikeMcQuaid** [comment]
@joshka Apologies, forgot about that!

**Comment by joshka** [comment]
No probs. will fix tomorrow

File: share/doc/homebrew/Tips-N'-Tricks.md (hunk 1 above)
Comment (joshka) [inline share/doc/homebrew/Tips-N'-Tricks.md]: https://github.com/josa42/atom-language-homebrew-formula/pull/4

Comment (MikeMcQuaid) [inline share/doc/homebrew/Tips-N'-Tricks.md]: Thanks! can you fix the package name here too and then we should be good to merge.

**Comment by joshka** [comment]
Squashed the commit that fixes the typo.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**